scalanet/src/io/iohk/scalanet/peergroup/UDPPeerGroup.scala

```diff
@@ -235,9 +236,9 @@ class UDPPeerGroup[M](val config: Config)(
 
     private def closeChannel(): Task[Unit] = {
       for {
-        _ <- Task.now(log.debug("Closing channel from {} to {}", localAddress: Any, remoteAddress: Any))
+        _ <- Task.delay(logger.debug("Closing channel from {} to {}", localAddress: Any, remoteAddress: Any))
```

Comment:
why not `Task(...)` ? It is a little shorter, and from what i remember task `apply` and `delay`, are same thing

Reply:
Yeah `apply`, `eval` and `delay` are all the same. I got used to writing [Sync[F].delay](https://typelevel.org/cats-effect/api/cats/effect/Sync.html#delay[A](thunk:=%3EA):F[A]) when using cats-effect, that's all. Can change it if you prefer the shorter version.

Reply:
yeah I would prefer using `apply` consistenlty